### PR TITLE
Update all examples to 10.0.3 

### PR DESCRIPTION
**Pull request by academo**

### CODE REVIEW

File: scripts/update-grafana-version.sh [hunk 1]
```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
```
Comment (academo) [inline scripts/update-grafana-version.sh:1]: The updates to package.json, docker and plugin.json files were made by this script.

File: scripts/update-grafana-version.sh [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                break
+            fi
+        done
+
+        if $upgradeFlag; then
+            upgradeCommand+="$dep@$GRAFANA_VERSION_TARGET "
+        else
+            upgradeCommand+="$dep@latest "
+        fi
+    done
```
Comment (xnyo) [inline scripts/update-grafana-version.sh:53]: Nit: It looks like this block is indented twice

**Review by xnyo — APPROVED** [review]
LGTM! Awesome work! :rocket:

File: scripts/update-grafana-version.sh (hunk 2 above)
Comment (academo) [reply scripts/update-grafana-version.sh:53]: better safe than sorry!

**Pull request merged**